components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/dto/APIDTO.java

```diff
@@ -350,6 +352,19 @@ public void setLabels(List<LabelDTO> labels) {
   }
 
   
+  /**
+   * The environment list configured with non empty endpoint URLs for the particular API.
+   **/
+  @ApiModelProperty(value = "The environment list configured with non empty endpoint URLs for the particular API.")
+  @JsonProperty("environmentList")
+  public List<String> getEnvironmentList() {
+    return environmentList;
+  }
+  public void setEnvironmentList(List<String> environmentList) {
+    this.environmentList = environmentList;
+  }
+
+  
```

Comment:
two empty lines

Reply:
The code of APIDTO.java is auto generated using swagger2cxf-maven-plugin [1]. Hence the empty lines were added by the plugin. Is it okay to manually modify the code?

[1] https://docs.google.com/document/d/1HzyWDfjqVc0KnENy6DOQMjgGmYaoHEnr-nZJbmEU24E/edit#

Reply:
If it is auto generated, we can leave it.